packages/android_intent/example/test_driver/android_intent_e2e.dart

```diff
@@ -49,4 +49,13 @@ void main() {
           e.message.contains('No Activity found to handle Intent');
     }));
   }, skip: !Platform.isAndroid);
+
+  testWidgets('#canResolve throws when no Activity is found',
```

Comment:
Could you add a success test here? Maybe you could check for the main activity of the example app? It should be installed every e2e run.